Various templates corrections

hgouraud opened the pull request:
arbre-h7: missisn if(cgi)...
lex_utf8: missing fr for on (weekday, missing %t in married to
perso_utils: missing date_place in married_to
welcome: hack for ineffective <CR> in Safari

sagotch commented on hd/etc/welcome.txt:
```diff
@@ -157,7 +157,9 @@ fr: Les <strong>droits des magiciens</strong> sont actuellement <strong>suspendu
         </div>
-        <button type="submit" class="d-none btn btn-lg btn-outline-primary" title="[*search/case sensitive]0 [person/persons]0"><i class="fa fa-search fa-fw"></i></button>
+        <button type="submit"
+          style="position:absolute;margin-left:-1000px" %(hack for Safari %)
```
Cela mérite des explications.

hgouraud committed:
Various templates corrections

arbre-h7: missisn if(cgi)...
lex_utf8: missing fr for on (weekday, missing %t in married to
welcome: hack for ineffective <CR> in Safari

hgouraud commented:
The [with] correction in perso_utils.txt is addressed in PR #1010

hgouraud commented on hd/etc/perso_utils.txt:
```diff
@@ -491,7 +491,7 @@
     %elseif;are_engaged;[*engaged%t to:::date_place]0%nn;
     %elseif;is_no_sexes_check;[*relationship%t to:::date_place]0%nn;
-    %elseif;is_no_mention;[*with]%nn;
+    %elseif;is_no_mention;%apply;capitalize%with;date_place%end; [with]%nn;
```
solution proposée dans PR #1010
